### PR TITLE
Add support for TextMate 2

### PR DESCRIPTION
**Pull request by lowski**
Small changes but they made this package compatible with TextMate 2.

### CODE REVIEW

File: vue.YAML-tmLanguage [hunk 1]
```diff
@@ -174,7 +174,7 @@ patterns:
       '1': {name: punctuation.definition.tag.end.html}
     end: (?=</(?i:style))
     patterns:
-    - include: source.css.scss
+    - include: source.scss
```
Comment (lowski) [inline vue.YAML-tmLanguage]: I believe it should be `source.scss` but can someone verify if it does not break on Sublime?

File: vue.tmLanguage [hunk 2]
```diff
@@ -581,7 +581,7 @@
 					<array>
 						<dict>
 							<key>include</key>
-							<string>source.css.scss</string>
+							<string>source.scss</string>
```
Comment (lowski) [inline vue.tmLanguage]: The same as above.

**Comment by skyronic** [comment]
Hey @lowski  

Thank you for the contribution. We use the `new` branch for the latest versions of sublime (using the new `sublime-syntax`). The `.tmLanguage` is only used for github syntax highlighting.

That said, if this is useful for Textmate 2 users, I think it'll be great to merge this, although I don't want to maintain/support anything on `master` since it's been out of date for a while. However, I need to ensure that this doesn't break compatibility for github syntax highlight.

**Comment by lowski** [comment]
Hi @skyronic , thanks for the info. I can create a new PR to the `new` branch.

> That said, if this is useful for Textmate 2 users, I think it'll be great to merge this

I use Textmate from time to time and it would be great to have syntax highlight for vue files. Even if it's not perfect, it's still better than nothing.

> However, I need to ensure that this doesn't break compatibility for github syntax highlight.

If I make a PR to the `new` branch it should be fine as there are no `.tmLanguage` files in the repo. WDYT?

**Comment by skyronic** [comment]
Hey, I was looking a bit deeper, I remember there was a lot of discussion about `source.scss` v/s `source.css.scss` because of a change adopted the most popular Sublime SCSS Extension (See #161) 

From what I can see, the current `tmLanguage` files works fine, and if it's broken in Textmate, then your patch might make it incompatible with older versions of Sublime and the github syntax highlighter.

Considering that we haven't gotten any requests for a Textmate version, I don't think I can merge any changes to the `tmLanguage` files at the moment, I apologize. 

If you wish to maintain a fork with textmate support in a personal repo, I'd be happy to redirect any users who ask for textmate support.

**Pull request closed without merging**